README.md

```diff
@@ -1 +1,53 @@
-# HelloMicro
\ No newline at end of file
+# Project: Hello Microverse
+
+> For the first milestone in building your hello microverse.
+
+> set up a new repository and prepare it for development using best practices (e.g. linters).
+
+
+# Screenshot
+
+>
+
+## Built with
+
+- HTML & CSS
+
+## Link to the live page
+
+> Feel free to check the []().
+
+## Authors
+
+👤 **Tapish Khari**
+
+- GitHub: [@TapishKhari](https://github.com/TapishKhari)
+- LinkedIN:[@TapishKhari](https://www.linkedin.com/in/tapish-khari-87a12855/)
+## Collaborators
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Getting Started
+
+To get a local copy follow these simple steps:
+
+- Copy this [link](git@github.com:TapishKhari/HelloMicro.git)
+
+- I built a branch feature.
+
+- On branch feature I created a HTML file.
+
+- Inserted linter on my HTML file 
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
### MIT.md file :spiral_notepad:
![mitfile (copy)](https://user-images.githubusercontent.com/93039359/186214250-99a92579-8752-4208-9c5d-1084ce85421d.png)

- [ ] Its good that you have a link to your MIT.md file but it doesn't exist in your repository :smile: please add it using this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md). Since, its the license file of your project so please take seriously :thinking:
- [ ] Please add your **name**, **current year**, **app type** in your MIT.md file. This is the license file of your project so please take it seriously :thinking:

Reply:
@HAMMAS-SALEEM : I have added and updated MIT.md file